src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_COUNT = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        int number = random.nextInt(MAX_BALL_COUNT + 1);
+        return new Ball(color, number);
```

Comment:
https://github.com/mate-academy/jv-oop-advanced/pull/740#pullrequestreview-1137947989
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_BALL_COUNT));
```